homeassistant/components/openweathermap/const.py

```diff
@@ -266,7 +267,7 @@
         name="Condition",
     ),
     SensorEntityDescription(
-        key=ATTR_FORECAST_NATIVE_PRECIPITATION,
+        key=ATTR_FORECAST_PRECIPITATION,
```

Comment:
Side note: The sensor entity descriptions should preferably be moved to the sensor platform and not use a constant for the key if it's only used in one place.